cmd/oklog/store.go

```diff
@@ -38,25 +40,50 @@ var (
-		segmentBufferSize        = flagset.Int64("store.segment-buffer-size", defaultStoreSegmentBufferSize, "per-segment in-memory read buffer during queries")
-		segmentReplicationFactor = flagset.Int("store.segment-replication-factor", defaultStoreSegmentReplicationFactor, "how many copies of each segment to replicate")
-		segmentRetain            = flagset.Duration("store.segment-retain", defaultStoreSegmentRetain, "retention period for segment files")
-		segmentPurge             = flagset.Duration("store.segment-purge", defaultStoreSegmentPurge, "purge deleted segment files after this long")
-		uiLocal                  = flagset.Bool("ui.local", false, "ignore embedded files and go straight to the filesystem")
-		filesystem               = flagset.String("filesystem", defaultFilesystem, "real, virtual, nop")
-		clusterPeers             = stringslice{}
+		debug                     = flagset.Bool("debug", false, "debug logging")
+		apiAddr                   = flagset.String("api", defaultAPIAddr, "listen address for store API")
+		clusterBindAddr           = flagset.String("cluster", defaultClusterAddr, "listen address for cluster")
+		clusterAdvertiseAddr      = flagset.String("cluster.advertise-addr", "", "optional, explicit address to advertise in cluster")
+		storePath                 = flagset.String("store.path", defaultStorePath, "path holding segment files for storage tier")
+		segmentConsumers          = flagset.Int("store.segment-consumers", defaultStoreSegmentConsumers, "concurrent segment consumers")
+		segmentTargetSize         = flagset.Int64("store.segment-target-size", defaultStoreSegmentTargetSize, "try to keep store segments about this size")
+		segmentTargetAge          = flagset.Duration("store.segment-target-age", defaultStoreSegmentTargetAge, "replicate once the aggregate segment is this old")
+		segmentDelay              = flagset.Duration("store.segment-delay", defaultStoreSegmentDelay, "request next segment files after this delay")
+		segmentBufferSize         = flagset.Int64("store.segment-buffer-size", defaultStoreSegmentBufferSize, "per-segment in-memory read buffer during queries")
+		segmentReplicationFactor  = flagset.Int("store.segment-replication-factor", defaultStoreSegmentReplicationFactor, "how many copies of each segment to replicate")
+		segmentRetain             = flagset.Duration("store.segment-retain", defaultStoreSegmentRetain, "retention period for segment files")
+		segmentPurge              = flagset.Duration("store.segment-purge", defaultStoreSegmentPurge, "purge deleted segment files after this long")
+		segmentPurgeDCS           = flagset.Bool("store.segment-purge-dcs", true, "move trashed segments to Storj DCS for cold storage on purge")
+		segmentPurgeDCSAccess     = flagset.String("store.segment-purge-dcs-access", "", "serialized access grant string for Storj DCS")
```

Comment:
Maybe this could default to `os.GetEnv("ACCESS_GRANT")` so you can choose to set the grant in env instead of passing it in as a flag? As part of the retrieval command I did this, finding it from [a libuplink usage example](https://github.com/storj/uplink/blob/main/examples/walkthrough/main.go#L85) :)

Reply:
Good call; thanks, Sean!